.github/workflows/build.yml

```diff
@@ -69,8 +69,49 @@ jobs:
         with:
           java-version: "adopt@1.11.0-11"
 
+      - name: Build and run unit tests
+        run: sbt "project server" test
+
+      - name: Prepare testing database
+        run: sbt flywayMigrate
+
+      - name: Build and run integration tests
+        run: sbt "project server" testIT doc
```

Comment:
Could these be run in parallel?

Reply:
It can be separated into different jobs or yml files.
In the case of the yml file, they will be running in parallel.

The current solution is a "hidden" check, which confirms that unit tests have no dependency on DB existence.
This idea is based on real bug reported in a balta.